css-view-transitions-1/Overview.bs

```diff
@@ -956,8 +956,12 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. Let |document| be [=this's=] [=relevant global object's=] [=associated document=].
 
+		1. If |document|'s [=Document/visibility state=] is "<code>hidden</code>",
```

Comment:
We need to do the same thing if the state flips to hidden while a transition was running too.

Reply:
Yes, that would be in the HTML spec.